src/Directory.Build.targets

```diff
@@ -1,6 +1,7 @@
 <Project>
 
   <PropertyGroup>
+    <PackageValidationBaselineVersion Condition="'$(MinVerTagPrefix)' == 'core-'">$(OTelLatestStableVer)</PackageValidationBaselineVersion>
```

Comment:
Add a comment describing why unlike other properties this needs to be in `Directory.Build.targets` instead of `Directory.Packages.props`

Reply:
Updated